internal/manifests/collector/container.go

```diff
@@ -91,6 +92,18 @@ func Container(cfg config.Config, logger logr.Logger, otelcol v1beta1.OpenTeleme
 			})
 	}
 
+	if cfg.CertManagerAvailability() == certmanager.Available {
+		volumeMounts = append(volumeMounts,
+			corev1.VolumeMount{
+				Name:      naming.TAClientCertificate(otelcol.Name),
+				MountPath: "/tls"},
+			corev1.VolumeMount{
+				Name:      "shared-ca-certificates",
+				MountPath: "/etc/ssl/certs/ca-certificates.crt",
+				SubPath:   "ca-certificates.crt",
+			})
```

Comment:
Like I mentioned in my comment below, I don't think this is necessary at all. For the specific mTLS between the collector and target allocator, we can use the CA certificates from the Secret itself.

Reply:
We need to append the CA we are creating for the mTLS connection to the trusted CA store in the image. That is in order for clients that use that store (Ours does) to trust our newly created CA.
The existing CA certs needs to be there for the collector to trust external certs, for the various vendors for example. 
The way i was thinking of achieving this is using the certificate from the secret, appending it to the trusted CA store that comes with the alpine docker image (the same one that is used in the original Collector's image) using that in place of the existing `/etc/ssl/certs/ca-certificates.crt`. 
As i mentioned in a previous comment, it is advisable to let `update-ca-certificates` do the appending as opposed to editing the /etc/ssl/certs/ca-certificates.crt file directly.
Im open to suggestions on how to append the new CA cert to `/etc/ssl/certs/ca-certificates.crt`

Reply:
Does setting `ca_file` in the configuration to the `ca.crt` from the Certificate Secret not achieve this? My understanding is that doing this causes the client to use that CA cert instead of the system root one, which is what we want here.

Reply:
I just tested that and it works. I must have had a different test environment set up where something let me to think it was not used in the client.
I will remove the whole CA trust store configs i added. Thanks